<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StrategyManagerWindow</class>
 <widget class="QWidget" name="StrategyManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Strategy Manager</string>
  </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: #ffffff;
    color: #1e293b;
    font-family: 'Segoe UI', system-ui, -apple-system, sans-serif;
}

QFrame#HeaderFrame {
    background-color: #ffffff;
    border-bottom: 1px solid #e2e8f0;
    padding: 10px;
}

QLabel#TitleLabel {
    font-size: 20px;
    font-weight: 800;
    color: #0f172a;
    letter-spacing: -0.5px;
}

QLabel#SummaryLabel {
    font-size: 12px;
    color: #64748b;
    font-weight: 500;
}

QPushButton#createButton {
    background-color: #2563eb;
    color: white;
    border: none;
    border-radius: 4px;
    font-weight: 700;
    font-size: 12px;
    padding: 8px 20px;
}
QPushButton#createButton:hover { background-color: #1d4ed8; }
QPushButton#createButton:pressed { background-color: #1e40af; }

QPushButton#buildCustomButton {
    background-color: #7c3aed;
    color: white;
    border: none;
    border-radius: 4px;
    font-weight: 700;
    font-size: 12px;
    padding: 8px 20px;
}
QPushButton#buildCustomButton:hover { background-color: #6d28d9; }
QPushButton#buildCustomButton:pressed { background-color: #5b21b6; }

QPushButton#deployTemplateButton {
    background-color: #0e7490;
    color: white;
    border: none;
    border-radius: 4px;
    font-weight: 700;
    font-size: 12px;
    padding: 8px 20px;
}
QPushButton#deployTemplateButton:hover { background-color: #0891b2; }
QPushButton#deployTemplateButton:pressed { background-color: #0c4a6e; }

QPushButton {
    background-color: #ffffff;
    color: #475569;
    border: 1px solid #cbd5e1;
    border-radius: 4px;
    padding: 6px 16px;
    font-size: 11px;
    font-weight: 600;
}
QPushButton:hover {
    background-color: #f8fafc;
    border-color: #94a3b8;
    color: #1e293b;
}
QPushButton:pressed {
    background-color: #f1f5f9;
}

/* Status Filter Buttons */
QPushButton#filterAllBtn, QPushButton#filterRunningBtn, 
QPushButton#filterPausedBtn, QPushButton#filterStoppedBtn {
    background-color: #ffffff;
    border: 1px solid #cbd5e1;
    border-radius: 0px;
    margin: 0px;
    color: #64748b;
    padding: 6px 14px;
    font-weight: 600;
}
QPushButton#filterAllBtn { border-top-left-radius: 4px; border-bottom-left-radius: 4px; border-right: none; }
QPushButton#filterStoppedBtn { border-top-right-radius: 4px; border-bottom-right-radius: 4px; border-left: none; }

QPushButton#filterRunningBtn, QPushButton#filterPausedBtn { border-right: none; }

QPushButton#filterAllBtn:checked, QPushButton#filterRunningBtn:checked, 
QPushButton#filterPausedBtn:checked, QPushButton#filterStoppedBtn:checked {
    background-color: #2563eb;
    color: white;
    border-color: #2563eb;
}

QTableView {
    background-color: #ffffff;
    gridline-color: #f1f5f9;
    border: none;
    selection-background-color: #f0f9ff;
    selection-color: #1e40af;
    alternate-background-color: #f8fafc;
    color: #334155;
    font-size: 12px;
}

QHeaderView::section {
    background-color: #f8fafc;
    color: #64748b;
    padding: 10px 12px;
    border: none;
    border-bottom: 2px solid #e2e8f0;
    border-right: 1px solid #f1f5f9;
    font-weight: 700;
    font-size: 10px;
    text-transform: uppercase;
}

QComboBox {
    background-color: #ffffff;
    color: #1e293b;
    border: 1px solid #cbd5e1;
    border-radius: 4px;
    padding: 5px 10px;
    font-size: 11px;
}
QComboBox:hover { border-color: #3b82f6; }
QComboBox::drop-down { border: none; width: 24px; }

QScrollBar:vertical {
    border: none;
    background: #f8fafc;
    width: 8px;
    margin: 0px;
}
QScrollBar::handle:vertical {
    background: #e2e8f0;
    border-radius: 4px;
    min-height: 20px;
    margin: 2px;
}
QScrollBar::handle:vertical:hover { background: #cbd5e1; }</string>
   </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="HeaderFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="headerLayout">
      <item>
       <layout class="QVBoxLayout" name="titleArea">
        <item>
         <widget class="QLabel" name="TitleLabel">
          <property name="text">
           <string>Strategy Manager</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="SummaryLabel">
          <property name="text">
           <string>Total: 0 | Running: 0 | Total P&amp;L: 0.00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="headerSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="createButton">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Create New Strategy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buildCustomButton">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>⚡ Build Custom</string>
        </property>
        <property name="toolTip">
         <string>Build a custom strategy using the visual builder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deployTemplateButton">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>� Manage Templates</string>
        </property>
        <property name="toolTip">
         <string>Browse, create, edit, clone, delete, and deploy strategy templates</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="filterLayout">
     <property name="leftMargin">
      <number>15</number>
     </property>
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>15</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Strategy Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="strategyFilterCombo">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="filterSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Filter by Status:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="filterAllBtn">
       <property name="text">
        <string>All</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="filterRunningBtn">
       <property name="text">
        <string>Running</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="filterPausedBtn">
       <property name="text">
        <string>Paused</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="filterStoppedBtn">
       <property name="text">
        <string>Stopped</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="actionLayout">
     <property name="leftMargin">
      <number>15</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>15</number>
     </property>
     <property name="bottomMargin">
      <number>15</number>
     </property>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pauseButton">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resumeButton">
       <property name="text">
        <string>Resume</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="actionSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="modifyButton">
       <property name="text">
        <string>Modify</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
